### ISSUE TITLE
add_unitcell() shows the same frame in viewer

### ISSUE THREAD

**Issue by ArtVoro** [arose/nglview#742]
version: nglview 1.1.3 and python 2.7
issue: when adding the unitcell via .add_unitcell() the trajectory viewer freezes (i.e. shows the same frame although the slider moves when pressing play button). 

code example:

```python
import MDAnalysis as mda
import nglview as ngl

u = mda.Universe("topology.top", "trajectory.xtc")
view = ngl.show_mdanalysis(u)
view.add_cartoon()
view.color_by('residueindex')
view.add_unitcell()
view
```

however, i can still move and rotate the object, which will refresh the conformation as long as im doing these actions.

**Comment by hainm** [comment]
I confirm this is a bug. Thanks for your report.

**Comment by hainm** [comment]
@arose 

The error is caused by below lines if a structure has `unitcell` representation (ok if not have it).

https://github.com/arose/nglview/blob/a9fea20074a6867f69339566264bbc43b85fa226/js/src/widget_ngl.js#L758-L760

Any idea how to fix it? thanks.

**Comment by hainm** [comment]
Console log:

<img width="993" alt="screen shot 2018-10-27 at 6 03 00 pm" src="https://user-images.githubusercontent.com/4451957/47609737-9478a080-da12-11e8-8e5a-f9e96d10387a.png">

**Comment by hainm** [comment]
uhm, I am not actually sure. If a structure has a `unitcell`, the coordinates of the structure won't be updated. May be a bug in NGL?

**Comment by hainm** [comment]
Example `parser/nctraj` won't work (the trajectory player was frozen if adding unitcell).

http://nglviewer.org/ngl/

![player](https://user-images.githubusercontent.com/4451957/47609845-ecb0a200-da14-11e8-8f47-3cab1ffff9a9.gif)

**Comment by hainm** [comment]
should be fixed in #769. Thanks for your report.
